bigquery_etl/glam/templates/clients_daily_histogram_aggregates_v1.sql

```diff
@@ -0,0 +1,85 @@
+{{ header }}
+WITH extracted AS (
+  SELECT
+    *,
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    REPLACE(ping_info.ping_type, "_", "-") AS ping_type,
+    COALESCE(
+      SAFE_CAST(SPLIT(client_info.app_display_version, '.')[OFFSET(0)] AS INT64),
+      0
+    ) AS app_version,
+    client_info.os AS os,
+    client_info.app_build AS app_build_id,
+    client_info.app_channel AS channel
+  FROM
+    `moz-fx-data-shared-prod.{{ source_table }}`
+  WHERE
+    DATE(submission_timestamp) = {{ submission_date }}
+    AND client_info.app_channel IN ("release", "fenixProduction")
+    AND client_info.client_id IS NOT NULL
+),
+histograms AS (
+  SELECT
+    {{ attributes }},
+    ARRAY<
+      STRUCT<
+        metric STRING,
+        metric_type STRING,
+        sum INT64,
+        value ARRAY<STRUCT<key STRING, value INT64>>
+      >
+    >[{{ histograms }}] AS metadata
+  FROM
+    extracted
+),
+flattened_histograms AS (
+  SELECT
+    {{ attributes }},
+    metadata.*
+  FROM
+    histograms,
+    UNNEST(metadata) as metadata
+  WHERE
+    value IS NOT NULL
+),
+-- ARRAY_CONCAT_AGG may fail if the array of records exceeds 20 MB when
+-- serialized and shuffled. This may exhibit itself in a pathological case where
+-- the a single client sends *many* pings in a single day. However, this case
```

Comment:
`ARRAY_CONCAT` is used here, instead of unnested and grouping everything together again, which is a departure from the telemetry version. This works because each client doesn't send many metric pings.